INSTRUCTION.md

```diff
@@ -0,0 +1,89 @@
+## Deploy the application following the commands:
+```
+cd ./.infrastructure/  
+kubectl apply -f namespace.yml
+kubectl apply -f busybox.yml
+kubectl apply -f clusterip.yml
+kubectl apply -f nodeport.yml
+kubectl apply -f deployment.yml
+kubectl apply -f hpa.yml
+kubectl apply -f todoapp-pod.yml
```

Comment:
Ensure that the file 'todoapp-pod.yml' exists and is necessary for your deployment. Typically, deploying a pod directly is not required if you are using a Deployment resource.